WordPress/src/test/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelperTest.kt

```diff
@@ -244,6 +245,13 @@ class PostListItemUiStateHelperTest {
         assertThat(state.data.showOverlay).isTrue()
     }
 
+    @Test
+    fun `show only delete and move to draft buttons on trashed posts`() {
+        val state = createPostListItemUiState(post = createPostModel(POST_STATE_TRASHED))
+        assertThat(state.actions[0].buttonType).isEqualTo(PostListButtonType.BUTTON_DELETE)
+        assertThat(state.actions[1].buttonType).isEqualTo(PostListButtonType.BUTTON_MOVE_TO_DRAFT)
```

Comment:
We might also want to check the size == 2. Otherwise, the test might succeed if more actions were available.

Reply:
Added in 577e191